certdb/pg/pg_test.go

```diff
@@ -0,0 +1,225 @@
+// +build postgresql
+
+package pg
+
+import (
+	"testing"
+	"time"
+
+	"github.com/cloudflare/cfssl/certdb"
+	"github.com/cloudflare/cfssl/certdb/testdb"
+)
+
+var (
+	db = testdb.PostgreSQLDB()
+)
+
+func roughlySameTime(t1, t2 time.Time) bool {
+	episilon := time.Second
+	if t1.After(t2) {
+		return t1.Sub(t2) < episilon
+	}
+	return t2.Sub(t1) < episilon
```

Comment:
yes, I will fix it. Initial thought is Math.Abs is for float64 while Duration is int64, converting int64 to float64 introduces loss of precisions. In this case it is not a problem since the difference is usually small enough that there is no loss.